db/migrate/20190107151222_add_omniauthinfo_to_accounts.rb

```diff
@@ -0,0 +1,17 @@
+class AddOmniauthinfoToAccounts < ActiveRecord::Migration
+  def change
+  	add_column :accounts, :omniauth_info, :text
+  	add_column :accounts, :uid, :string
+  	add_column :accounts, :provider, :string
```

Comment:
I think we should add an index for this `provider` column.

Reply:
We should add an index for any column that will be used for search